### PR TITLE
Bump devon_rex images from 2.45.4 to 2.45.9

### PR DESCRIPTION
**Pull request by dev-yakuza**
This change is created by:
```
$ bundle exec rake bump:devon_rex'[2.45.9]'
```

See also:
- https://github.com/sider/devon_rex/releases/tag/2.45.9
- https://github.com/sider/devon_rex/blob/2.45.9/CHANGELOG.md

### Checklist

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ All notable changes to this project will be documented in this file.
 Updated tools:
 
 - **Brakeman** 5.0.4 -> 5.1.1 [#2555](https://github.com/sider/runners/pull/2555)
+- **Bundler** 2.2.24 -> 2.2.26 [#2632](https://github.com/sider/runners/pull/2632)
 - **Checkstyle** 8.44 -> 8.45.1 [#2599](https://github.com/sider/runners/pull/2599)
 - **detekt** 1.17.1 -> 1.18.0 (**breaking**) [#2616](https://github.com/sider/runners/pull/2616)
+- **devon_rex** 2.45.4 -> 2.45.9 [#2632](https://github.com/sider/runners/pull/2632) ([diff](https://github.com/sider/devon_rex/compare/2.45.4...2.45.9))
```
Comment (ybiquitous) [inline CHANGELOG.md:15]: Please add a new section "Updated environments:" for devon_rex and Bundler.
See https://github.com/sider/runners/blob/master/CHANGELOG.md#0530

**Comment by dev-yakuza** [comment]
@ybiquitous (cc @radomirbosak)
Some logic about `peerDependencies` is changed in `npm`

- Related: https://github.com/npm/cli/releases/tag/v7.20.3

Screen Shot 2021-08-23 at 11 53 34

So, I got an error like the below.

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: eslint-config-airbnb@18.2.1
npm ERR! Found: eslint-plugin-import@2.24.0
npm ERR! node_modules/eslint-plugin-import
npm ERR!   eslint-plugin-import@"2.24.0" from the root project
npm ERR!   peer eslint-plugin-import@"^2.22.1" from eslint-config-airbnb@18.2.1
npm ERR!   node_modules/eslint-config-airbnb
npm ERR!     eslint-config-airbnb@"18.2.1" from the root project
npm ERR!   1 more (eslint-config-react-app)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! eslint-config-airbnb-base@"^14.2.1" from eslint-config-airbnb@18.2.1
npm ERR! node_modules/eslint-config-airbnb
npm ERR!   eslint-config-airbnb@"18.2.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: eslint-plugin-import@2.24.1
npm ERR! node_modules/eslint-plugin-import
npm ERR!   peer eslint-plugin-import@"^2.22.1" from eslint-config-airbnb-base@14.2.1
npm ERR!   node_modules/eslint-config-airbnb-base
npm ERR!     eslint-config-airbnb-base@"^14.2.1" from eslint-config-airbnb@18.2.1
npm ERR!     node_modules/eslint-config-airbnb
npm ERR!       eslint-config-airbnb@"18.2.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --no-strict-peer-deps, --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/jeonghean/.npm/eresolve-report.txt for a full report.
```

I think it's not a bug on `npm`. 🤔  maybe our `package.json` is wrong.

- Releated: https://github.com/npm/cli/issues/3616

So, I just update the `eslint-plugin-import` version like the below. The error is gone on local.

```diff
- "eslint-plugin-import": "2.24.0"
+ "eslint-plugin-import": "2.24.1"
```

So, I'll merge the PR(https://github.com/sider/runners/pull/2625) And I will try this again.

**Comment by ybiquitous** [comment]
@dev-yakuza I could confirm HEAD on `master` works locally! 👍🏼

**Comment by dev-yakuza** [comment]
@ybiquitous
Could you approve this PR?

**Review by ybiquitous — APPROVED** [review]
Great! 👏🏼

**Pull request merged**